src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private String[] colors = {"red", "blue", "green", "yellow", "pink", "white", "black"};
```

Comment:
Consider using an enum for the colors instead of a String array. Enums provide a more structured and safer way to handle predefined constants, and you can easily get the name of the enum constant using the `name()` method.